pages/index.js

```diff
@@ -1,9 +1,11 @@
+import Button from "../components/ui/Button";
+
 export default function Home() {
   return (
-    <div className="flex h-screen">
-      <h1 className="m-auto font-Neufile text-6xl font-bold  text-red-900  underline ">
-        Hello Maroon!
-      </h1>
+    <div className="">
+    
+     
+      <Button text="testing" styles=""/>
     </div>
```

Comment:
This page should only be used for testing since this is a reusable component. so after the test is done, this page has to be returned to its initial state.
for more details any question is welcome.

Reply:
I removed it